core/src/test/scala/integration/kafka/server/GssapiAuthenticationTest.scala

```diff
@@ -92,15 +92,16 @@ class GssapiAuthenticationTest extends IntegrationTestHarness with SaslSetup {
    * Tests that Kerberos replay error `Request is a replay (34)` is not handled as an authentication exception
    * since replay detection used to detect DoS attacks may occasionally reject valid concurrent requests.
    */
-  @Test
-  def testRequestIsAReplay(): Unit = {
+  @ParameterizedTest
+  @ValueSource(strings = Array("kraft"))
+  def testRequestIsAReplay(quorum: String): Unit = {
```

Comment:
Why is `quorum` an input parameter to this test? Seems the value would always be `"kraft"`? -- I also also don't see where `quorum` is actually used?

Reply:
While the parameter isn't used directly, it affects the test name which some code in `TestUtils.scala` uses to determine if the test is kraft or not. This is standard for our kraft tests